test/test_files/function/list.test

```diff
@@ -2254,3 +2254,12 @@ a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a11*a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a12*a0eebc
 -STATEMENT RETURN [[23, 432], [], [NULL]];
 ---- 1
 [[23,432],[],[]]
+
+-CASE listContainsCast
```

Comment:
I'd move this test up to closer to `ListIn`. Also, can we break this test file to more test cases? it looks horrible one test case with more than 2000 lines.